addons/mail/static/src/js/models/threads/dm_chat.js

```diff
@@ -49,6 +51,28 @@ var DMChat = TwoUserChannel.extend({
         return this._directPartnerID;
     },
     /**
+    * Get the out of office info
+    *
+    * @returns {string}
+    */
+    getOutOfOfficeInfo: function () {
+        if (this.getStatus().indexOf('leave') === -1) {
+            return undefined;
+        }
+        var date = moment(this._outOfOfficeDateEnd);
+        var formated_date = date.format('ll');
+        if (moment().format('ll') === formated_date) {
+            formated_date = date.format("HH:mm");
+        } else {
+            var current_year = (new Date()).getFullYear();
+            if (formated_date.endsWith(current_year)) { // Dummy logic to remove year (only if current year), we will maybe need to improve it
+                formated_date = formated_date.slice(0, -4);
```

Comment:
this does not work with some languages for which `moment.format('LL')` starts with the full year

Reply:
```js
date.toLocaleDateString(window.navigator.language, { day: 'numeric', month: 'short' });
```